README.md

```diff
@@ -22,7 +22,7 @@
 
 ### Dependencies
 ```
-git clone https://github.com/nodejs/website.git
+git clone https://github.com/nodejs/iojs.org
```

Comment:
Oh, sorry. I'll fix it. Anyway, both url's resolve to the same place... do you know if that mistake could be a problem anywhere else?